### PR TITLE
First pass for the stash origin documentation (SOFTWARE-3900)

### PR DESCRIPTION
**Pull request by efajardo**

### CODE REVIEW

File: docs/data/stashcache/run-stash-origin-container.md [hunk 1]
```diff
@@ -0,0 +1,123 @@
...
+
+```
+set StashOriginPublicAuthfile = /etc/xrootd/public-origin-authfile
+```
+
+
+### Disabling OSG monitoring ###
+
+By default, XCache reports to the OSG so that OSG staff can monitor the health of data federations.
+If you would like to report monitoring information to another destination, you can disable the OSG monitoring by setting
```
Comment (matyasselmeci) [inline docs/data/stashcache/run-stash-origin-container.md:49]: Do you _have to_ disable OSG reporting to report somewhere else, or can you report to two places?

Comment (efajardo) [reply docs/data/stashcache/run-stash-origin-container.md:49]: Yes you can but that is kinda advanced. So it should be either monitoring or nothing else

Comment (matyasselmeci) [reply docs/data/stashcache/run-stash-origin-container.md:49]: So how about something like
```suggestion
To disable OSG monitoring (for example so you can report monitoring information to another destination), set
```

File: docs/data/stashcache/run-stash-origin-container.md [hunk 2]
```diff
@@ -0,0 +1,122 @@
...
+1. **Docker:** For the purpose of this guide, the host must have a running docker service and you must have the ability to start containers (i.e., belong to the `docker` Unix group).
+1. **Network ports:** The Stash Origin listens for incoming HTTP/S and XRootD connections on ports 1094 and 1095 (by default).
+1. **File Systems:** Stash Origin needs a partition on the host to store user data.
+
+Configuring Stash Origin
+------------------------
+
+In addition to the required configuration above (ports and file systems), you may also configure the behavior of your origin with the following variables using an environment variable file:
+
+Where the environment file on the docker host, `/opt/origin/.env`, has (at least) the following contents (replace "YOUR_SITE_NAME" with the name of your site as [registered in Topology](/stashcache/install-origin/#registering-the-origin):
```
Comment (matyasselmeci) [inline docs/data/stashcache/run-stash-origin-container.md:25]:
```suggestion
Where the environment file on the docker host, `/opt/origin/.env`, has (at least) the following contents (replace "YOUR_SITE_NAME" with the name of your site as [registered in Topology](/data/stashcache/install-origin/#registering-the-origin):
```

**Pull request merged**